package.json

```diff
@@ -41,10 +41,12 @@
     "lint:fix": "yarn run lint --fix",
     "lint": "eslint --cache --ext js,ts,tsx,mdx .",
     "prebuild": "shx rm -rf dist",
-    "test:coverage": "yarn run test --coverage",
+    "test:unit:coverage": "yarn run test:unit --coverage",
     "test:watch": "yarn run test --watch",
     "test:update": "yarn run test --updateSnapshot",
     "test": "LC_ALL=en_US.UTF-8 jest",
+    "test:unit": "yarn test ./src/components ./src/helpers ./src/utils",
```

Comment:
Why specify folders ? 🤔

Reply:
To avoid running `ay11.test.tsx` in unit tests